src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -131,10 +131,12 @@ interfaces:
         - shelf
         - books
         - edition
+        - review_copy
```

Comment:
Thanks. Please also warn in the GAPIC codegen if a oneof field is marked required or flattened that we are ignoring required/flattened for that field.